test/verify/check-storage-vdo

```diff
@@ -110,7 +110,7 @@ class TestStorageVDO(StorageCase):
         # Delete
 
         b.click('.pf-c-card__header:contains("VDO") button:contains("Delete")')
-        self.confirm()
+        self.confirm_with_retry()  # device might still be busy, retry a bit
```

Comment:
Instead of blindly retrying we could also maybe check if some process is keeping the device busy.
So:
wait("lambda: self.machine.execute("lsof | grep -v /whatever"), delay=3)
At least we will know if it's busy and timeouts, that it failed in this command. What do you think?

Reply:
We could check the exact error message in the dialog and have a list of expected error messages.

Reply:
Sure that also makes sense!

Reply:
Done!